config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -117,6 +117,9 @@ presubmits:
         args:
           - runner.sh
           - "./scripts/ci-e2e.sh"
+        env:
+          - name: GINKGO_FOCUS
```

Comment:
I wonder if the `GINKGO_FOCUS=` [specified in `./scripts/ci-e2e.sh`](https://github.com/kubernetes-sigs/cluster-api/blob/9a1948ee0bc7226f26276d778b140630de88a04f/scripts/ci-e2e.sh#L54) will override the env var specified here.

In the `test/e2e/Makefile` we have a [conditional variable assignment operator](https://github.com/kubernetes-sigs/cluster-api/blob/9a1948ee0bc7226f26276d778b140630de88a04f/test/e2e/Makefile#L52) `GINKGO_FOCUS ?= ` which defaults to empty.

So we may not need the variable specified in `./scripts/ci-e2e.sh`.

Info on Makefile conditional variable assignment.
https://www.gnu.org/software/make/manual/make.html#index-_003f_003d

Reply:
Yes, it should be deleted in ci-e2e.sh.

Reply:
Yes, good catch @wfernandes .
Created a PR for this: https://github.com/kubernetes-sigs/cluster-api/pull/3583